From 1a66f120abddf36eaf2540532ddeb7f7767442c5 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Sat, 1 Feb 2020 08:58:11 +0000
Subject: [PATCH] ARM: dts: Remove CMA allocation from Pi 4 dts

The 5.5 tree includes a patch to disable the CMA command line
parameter and replace it with properties from a DT node.
The upstream Pi 4 .dts, now used downstream with modifications,
includes the "linux,cma" node, but only reserves 32MB which is
often not enough.

Temporarily remove the "linux,cma" node to reenable the command line
parameter.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711-rpi-4-b.dts | 4 ++++
 1 file changed, 4 insertions(+)

--- a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
@@ -167,6 +167,10 @@
 	};
 
 	/delete-node/ wifi-pwrseq;
+
+	reserved-memory {
+		/delete-node/ linux,cma;
+	};
 };
 
 &mmcnr {
